docker/docker-compose.yml

```diff
@@ -77,7 +77,7 @@ services:
       - ./logs/php:/var/log/php
       - ./bin:/var/scripts
     ports:
-      - "80:80"
+      - "${PORT_WORDPRESS:-80}:80"
     restart: always
     env_file:
       - default.env
```

Comment:
What do you think about adding `../.env` to the list here? Might be it's even picked automatically so it would be more for discoverability. If it's lowest, it has the highest priority if the same value is repeated in other files.

Reply:
Nah, screw that. That would make Docker fail if the file doesn't exist (https://github.com/Automattic/jetpack/pull/9317)

Reply:
>I guess I'd vote for not adding /.env (a.k.a ../.env.) to the containers at the moment, and to rethink it if there's a specific use case later (or if the current setup proves too confusing).

Agreed 👍  💯